ports/esp32/boards/ESP32_GENERIC_C2/mpconfigboard.cmake

```diff
@@ -0,0 +1,8 @@
+set(IDF_TARGET esp32c2)
+
+set(SDKCONFIG_DEFAULTS
+    boards/sdkconfig.base
+    ${SDKCONFIG_IDF_VERSION_SPECIFIC}
+    boards/sdkconfig.ble
+    boards/sdkconfig.26mhz
```

Comment:
Do all ESP32-C2 modules use 26MHz crystals? I see the docs say it can support both 26MHz and 40MHz.